priv/perf/apps/load_testing/lib/utils/nonce_tracker.ex

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTesting.Utils.NonceTracker do
+  @moduledoc """
+  Nonce tracker for sending Ethereum transactions
+  """
+  use GenServer
+
+  alias ExPlasma.Encoding
+
+  def start_link(init_state \\ %{}) do
```

Comment:
no need to do this, pass `args` all the way down to init, and if you don't need them there
just `def init(_) do`